client/src/Pages/EditPage/EditPage.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {useState, useEffect} from 'react'
+import { useNavigate } from 'react-router-dom';
+import './Edit.css'
+import Field from './Field.js'
+
+async function getUserdetails() {
+	var res = await fetch("/edit/getUser")
+	.then(res => res.json())
+	.then( data => {
+		return data
+	})
+
+}
+
+export default function EditPage(props) {
+    const [details, setDetails] = useState({});
+
+    useEffect(async () => {
+        var details = await fetch("/edit/getUser")
+        .then(res => res.json())
+        .then( data => {
+            setDetails(data)
+        })
+		//setDetails(details)
+
+    }, [])
```

Comment:
Hi, could you add some code to redirect if they aren't logged in? Or LMK if you want to save this for another PR